packages/exchange-analytics/test/unit/models.test.ts

```diff
@@ -0,0 +1,78 @@
+import { Provider } from '../../src/api/constants'
+import IFundingRate from '../../src/models/funding-rate'
+
+describe('models', () => {
+  describe('binance transformer', () => {
+    it('hydrates an IFundingRate object', async () => {
+      const getFundingRatePayload = {
+        symbol: 'ETHUSDT',
+        markPrice: '1461.20000000',
+        indexPrice: '1461.88822394',
+        estimatedSettlePrice: '1461.84099962',
+        lastFundingRate: '0.00007817',
+        interestRate: '0.00010000',
+        nextFundingTime: 1663344000000,
```

Comment:
Replace it by `1663344000001` and make sure `Math.floor()` logic works as expected